examples/testing/multi_frameworks_toolchain/example/Specs2ExampleTest.scala

```diff
@@ -3,9 +3,7 @@ package example
 import org.specs2.mutable.SpecWithJUnit
 
 class Specs2ExampleTest extends SpecWithJUnit {
-
-  "Example" in {
-    1 mustEqual 1
+  "Example" >> {
```

Comment:
Can you explain such changes?

Reply:
Yeah. I was writing comments, but I will here.

In specs2 v5 should is defined both as expect matched (like must) and as dsl. Unfortunately without specifying extended trait (ExtendedBlockDsl) it does not know which one to use and compile fails. I could not extend ExtendedBlockDsl, because in v4 of specs2 it does not exist.

Also, mustEqual is not present in v5, and I substituted it for another matcher.

Reply:
Regarding `in`, I had to extend the same DSL builder in some of the test classes, so I decided to align them with `should` replacements and just use >> which is cross-compatible. I could not debug exactly why, because the entire targets don't build on my intellij due to other errors...